fakeredis/_server.py

```diff
@@ -2490,12 +2490,22 @@ class FakeConnection(redis.Connection):
     description_format = "FakeConnection<db=%(db)s>"
 
     def __init__(self, server, db=0, password=None,
+                 socket_timeout=None, socket_connect_timeout=None,
+                 socket_keepalive=False, socket_keepalive_options=None,
+                 socket_type=0, retry_on_timeout=False,
```

Comment:
Are these in the same position/order that the redis `__init__` takes them?

Reply:
Yeah [at least as it is currently on master](https://github.com/andymccurdy/redis-py/blob/656b551eb247c491976646092a71320407c0fa03/redis/connection.py#L488).  I had forgotten a couple and just added them. Sorry.